packages/in_app_purchase/in_app_purchase_android/lib/src/in_app_purchase_android_platform.dart

```diff
@@ -249,7 +253,8 @@ class InAppPurchaseAndroidPlatform extends InAppPurchasePlatform {
   }
 
   Future<List<PurchaseDetails>> _getPurchaseDetailsFromResult(
-      PurchasesResultWrapper resultWrapper) async {
+    PurchasesResultWrapper resultWrapper,
```

Comment:
Why have you added commas everywhere? Changing formatting of lines that you aren't otherwise changing makes the diff (and git blame) more confusing.